project/src/main/java/com/google/sps/utilities/DatabaseService.java

```diff
@@ -0,0 +1,10 @@
+package com.google.sps.utilities;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.Spanner;
+
+public interface DatabaseService {
```

Comment:
I think it would be better if the interface provides query, write, close instead of spanner and client. So writer and reader of DatabaseWrapper don't need to know the details of interaction of DB.

Reply:
Ack, no longer applies - thank you for reviewing.